index.html

```diff
@@ -134,7 +134,7 @@ <h3>Enter the Size(N) you want for your matrix</h3>
                 </li>
                 <li>
                     <span class="icon">
-                        <img src="assets/icons/direct.svg" alt="placeHolder">
+                        <img class= "low_icon" src="assets/icons/direct.svg" alt="placeHolder">
```

Comment:
what does **low_icon** mean?

Reply:
some svg has more black color than the other (bolder).
so I reduced the contrast of those svg to make it looks better with other elements.